franklin/core/views.py

```diff
@@ -10,6 +11,22 @@
 
 @api_view(('GET',))
 @permission_classes((AllowAny, ))
+def dashboard_init(request):
+    """
+    For the dashboard to initialize itself with the api
+    ---
+    type:
+        client_id:
+            type: string
+            description: github client_id needed for social auth
+            required: true
+    """
+    message = {'client_id': os.environ['SOCIAL_AUTH_GITHUB_KEY']}
+    return Response(message, status=status.HTTP_200_OK)
```

Comment:
For now we aren't going to secure this endpoint. Any data this returns would be readable by inspecting the FE dashboard anyway.